src/jit/block.cpp

```diff
@@ -1375,15 +1375,15 @@ BasicBlock* Compiler::bbNewBasicBlock(BBjumpKinds jumpKind)
 }
 
 //------------------------------------------------------------------------
-// blockHasEHFlowIn: Determine if this block has incoming exception flow.
+// blockhasEHBoundaryIn: Determine if this block has an incoming EH boundary.
 //
 // Arguments:
 //    block   - the BasicBlock
```

Comment:
Nit: there are no arguments for this and the prev methods.

Reply:
Right, thanks. I previously had these in LSRA but realized that they really belong on `BasicBlock`.